src/main/webapp/app/course/manage/course-management-exercises.component.html

```diff
@@ -1,33 +1,100 @@
 <ng-container *ngIf="course">
-    <h3>
-        <span
-            >{{ course.title }} - {{ quizExercisesCount + programmingExercisesCount + modelingExercisesCount + fileUploadExercisesCount + textExercisesCount }}
-            {{ 'artemisApp.course.exercises' | artemisTranslate }}</span
-        >
-    </h3>
+    <div class="row">
+        <div class="col">
+            <h3>
+                <span>
+                    {{ course.title }}
+                    - {{ getExerciseCount() }}
+                    {{ 'artemisApp.course.exercises' | artemisTranslate }}
+                </span>
+            </h3>
+        </div>
+        <div class="col text-end align-self-center">
+            <button id="toggleSearchButton" class="btn btn-primary" type="button" (click)="toggleSearch()">
```

Comment:
In case you remove the reset on hide, you could set the variable `showSearch` here directly.